src/vector/v.scatterplot/v.scatterplot.py

```diff
@@ -144,14 +145,61 @@
 # % exclusive: color,rgbcolumn
 # %end
 
+# %option
+# % key: bins
+# % type: string
+# % label: 2D bins
+# % description: The number of bins in x and y dimension. Density is expressed as the number of points falling within the x and y boundaries of a bin.
+# % required: no
+# % answer: 30,30
+# % guisection: Density
+# %end
+
+# %option
+# % key: density_colormap
+# % type: string
+# % label: Density plot color map
+# % description: Select the color map to be used for the density scatter plot
+# % required: no
+# % answer: viridis
+# % options: viridis,plasma,inferno,magma,cividis,Greys,Purples,Blues,Greens,Oranges,Reds,YlOrBr,YlOrRd,OrRd,PuRd,RdPu,BuPu,GnBu,PuBu,YlGnBu,PuBuGn,BuGn,YlGn
```

Comment:
To avoid hardcoding the colors you may want to use the `G_OPT_M_COLR` macro.

Reply:
The color name is used to select a matplotlib colormap. The matplotlib colormaps do not match those from grass gis (not by name at least). I am inclined to leave it as it is now, but will look into how I can still leverage this macro option at another time.